S11-modules/rakudolib.t

```diff
@@ -0,0 +1,15 @@
+use v6;
+use Test;
+use lib $?FILE.IO.parent(2).add("packages/Test-Helpers");
+use Test::Util;
+
+plan 1;
+
+# Test deprecated environment variable RAKUDOLIB
+# TODO remove this file when RAKUDOLIB is removed
+is_run 'BEGIN { BEGIN { q{S11modulesRakuLibTest.rakumod}.IO.spurt(q{package { say q{all your base} }}); %*ENV<RAKULIB>=qq{}; }; use S11modulesRakuLibTest }',
```

Comment:
Once it's in roast it's specced. We should not spec RAKUDOLIB. Especially not in roast (which is independent of a concrete implementation, such as rakudo).
So I'd like to have this test removed.

Reply:
I'll remove the test. And I'll try to move it to Rakudo later if that's okay.

Reply:
Wait with moving until something concrete is decided in raku/problem-solving#167.

Reply:
Okay, it's removed from roast, but I have done nothing else.